### DISCUSSION TITLE
[Ollama Remote] How to set all parameters to model default?

### DISCUSSION THREAD

**Discussion by b0x3s** [Mobile-Artificial-Intelligence/maid#370] [Q&A]
Using the app, I can connect to my local Ollama server running on another machine. But I don't find a way to set all parameters to the default state (to run models with parameters straight from the modelfile). How can I set up Maid to do that?

**Comment by thanhminhmr** [comment]
I also got the same problem, @danemadsen will there be an option for this?

**Comment by danemadsen** [comment]
@b0x3s @thanhminhmr 
Ive added an option to use default ollama options. You can use a build with it here:
https://github.com/Mobile-Artificial-Intelligence/maid/actions/runs/8091949655

It will be included in the next full release also
